### PR TITLE
Sanity-check and cleanup of permissions, accross roles. DDFFORM-711 DDFFORM-625

### PR DESCRIPTION
**Pull request by rasben**

### CODE REVIEW

File: config/sync/user.role.administrator.yml [hunk 1]
```diff
@@ -190,6 +188,7 @@ permissions:
   - 'role-assign users by role'
   - 'schedule publishing of eventseries'
   - 'schedule publishing of nodes'
+  - 'select account cancellation method'
```
Comment (rasben) [inline config/sync/user.role.administrator.yml:191]: local_admin had this permission, but admin doesnt. seems like a mistake.

File: config/sync/user.role.administrator.yml [hunk 2]
```diff
@@ -86,7 +85,6 @@ permissions:
   - 'administer nodes'
   - 'administer openid connect clients'
   - 'administer orphaned events entities'
-  - 'administer paragraphs categories'
```
Comment (rasben) [inline config/sync/user.role.administrator.yml:89]: this just confused the editors, as they cant edit the paragraphs to assign categories.

File: config/sync/user.role.local_administrator.yml [hunk 3]
```diff
@@ -61,6 +60,7 @@ permissions:
   - 'access files overview'
   - 'access media overview'
   - 'access site in maintenance mode'
+  - 'access site reports'
```
Comment (rasben) [inline config/sync/user.role.local_administrator.yml:63]: this seems pretty harmless, and useful.

File: config/sync/user.role.external_system.yml [hunk 4]
```diff
@@ -5,14 +5,14 @@ dependencies:
...
     - rest
     - system
+    - toolbar
 id: external_system
 label: 'External system'
 weight: 7
 is_admin: null
 permissions:
-  - 'edit eventinstance entity'
+  - 'access toolbar'
```
Comment (rasben) [inline config/sync/user.role.external_system.yml:16]: external users needs access to the toolbar, to be able to edit their user password etc.

File: config/sync/user.role.mediator.yml [hunk 5]
```diff
@@ -100,11 +100,14 @@ permissions:
...
   - 'use text format basic'
   - 'use text format limited'
+  - 'view all media revisions'
   - 'view any unpublished content'
   - 'view own unpublished content'
   - 'view own unpublished media'
   - 'view scheduled content'
   - 'view scheduled eventseries'
+  - 'view scheduled media'
+  - 'view scheduled taxonomy_term'
```
Comment (kasperg) [inline config/sync/user.role.mediator.yml:110]: Do you have any comments about why these should be added?

**Review by kasperg — APPROVED** [review]
:+1: LGTM

I appreciate the comments you have added to this PR. I would have appreciated them even more if they were commit comments in separate commits 😉 .

File: config/sync/user.role.mediator.yml (hunk 5 above)
Comment (rasben) [reply config/sync/user.role.mediator.yml:110]: Because it matches other permissions, and i think they're missing by mistake

**Pull request merged**